datadog/resource_datadog_integration_pagerduty.go

```diff
@@ -18,10 +18,16 @@ func resourceDatadogIntegrationPagerduty() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
+			"individual_services": {
```

Comment:
Is the main reason we're enforcing this because if you do individual services you still need the main resource to configure global integration fields?

Reply:
I was wondering if there were a way to not rely on this resource later, but I'm seeing that it makes sense since we'll always have some global components.

Reply:
Yeah, we'll always need it to be able to:

* activate the integrtation in the first place
* configure global settings of the integrtation